cypress/e2e/mini-portfolio/activity-history.test.ts

```diff
@@ -93,24 +94,15 @@ describe('mini-portfolio activity history', () => {
   })
 
   it('should deduplicate activity history by nonce', () => {
-    cy.visit('/swap', { ethereum: 'hardhat' }).hardhat({ automine: false })
+    cy.visit(`/swap?inputCurrency=ETH&outputCurrency=${USDC_MAINNET.address}`, { ethereum: 'hardhat' }).hardhat({
+      automine: false,
+    })
 
     // Input swap info.
-    cy.get('#swap-currency-input .token-amount-input').clear().type('1')
-    cy.get('#swap-currency-output .open-currency-select-button').click()
-    cy.contains('USDC').click()
-    cy.get('#swap-currency-output .token-amount-input').should('not.equal', '')
+    cy.get('#swap-currency-input .token-amount-input').clear().type('1').should('have.value', '1')
+    cy.get('#swap-currency-output .token-amount-input').should('not.have.value', '')
 
-    // Set slippage to a high value.
-    cy.get(getTestSelector('open-settings-dialog-button')).click()
-    cy.get(getTestSelector('max-slippage-settings')).click()
-    cy.get(getTestSelector('slippage-input')).clear().type('5')
-    cy.get('body').click('topRight')
-    cy.get(getTestSelector('slippage-input')).should('not.exist')
```

Comment:
I added the slippage setting here to *avoid* flake I encountered related to network slippage--are you sure about this?

Reply:
I ran this about 10 times and the only flake I got was in swapping, _not_ here.